buyer_data/views.py

```
@@ -34,13 +37,13 @@ def buyer_data(r, buyer_username):
 
 @api_view(['GET', 'POST'])
 @permission_classes([IsAuthenticated])
-def get_bought_estate(r):
+def get_bought_estate(r, buyer_username):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)